media-gfx/alembic/metadata.xml

```diff
@@ -14,9 +14,25 @@
 	developed by Sony Pictures Imageworks Inc. and Lucasfilm Ltd.
 </longdescription>
 <use>
+	<flag name="arnold" restrict="&gt;=media-gfx/alembic-1.7.9">
```

Comment:
there is no need at all to restrict USE flag descriptions just because they do not exist in earler versions. Please only use this if the *meaning* of the same USE flag is changed between versions.